docs/monitoring.md

```diff
@@ -176,6 +176,7 @@ Each instance can report to zero or more _sinks_. Sinks are contained in the
 * `JmxSink`: Registers metrics for viewing in a JMX console.
 * `MetricsServlet`: Adds a servlet within the existing Spark UI to serve metrics data as JSON data.
 * `GraphiteSink`: Sends metrics to a Graphite node.
+* `Slf4jSink`: Sends metrics to log4j as logging entries
```

Comment:
Fair point. I'll change this to "Sends metrics to slf4j as log entries." to be exact.